Chess/include/Chess.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <iostream>
+#include <Windows.h>
+#include <string>
+
+using std::cout;
+using std::cin; 
+using std::endl;
+using std::string; 
+
+const int _SIZE = 21;
```

Comment:
Avoid using global variables: In the Chess class, the _SIZE variable is declared as a global variable. It's generally better to avoid global variables. Instead, you can make it a member variable of the Chess class.